src/framework/handlers/esmodule.js

```diff
@@ -50,8 +48,17 @@ class EsModuleHandler {
             };
         }
 
-        console.log(ASSET_BASE_URL + url.load);
-        import(ASSET_BASE_URL + url.load).then(({ default: module }) => {
+        /* eslint-disable */
+        /*#if _ASSET_BASE_URL
+        // The following code is neccesary to import ES modules
+
+        const finallUrl = $_ASSET_BASE_URL +  url.load;
+        //#else */
+        const finallUrl = url.load;
+        //#endif
+        /* eslint-enable */
+
+        import(finallUrl).then(({ default: module }) => {
```

Comment:
No time for testing rn, but I believe this may make `webpack` grumpy again: https://github.com/playcanvas/engine/pull/5737